hls4ml/templates/vivado_template.py

```diff
@@ -375,6 +407,7 @@ def __init__(self):
         self.register_templates('BatchNormalization'     , batchnorm_function_template,   batchnorm_config_template, batchnorm_include_list)
         self.register_templates('Conv1D'                 , conv1d_function_template,      [conv1d_config_template, conv_mult_config_template], conv1d_include_list)
         self.register_templates('Conv2D'                 , conv2d_function_template,      [conv2d_config_template, conv_mult_config_template], conv2d_include_list)
+        self.register_templates('Conv2DBatchnorm'        , conv2dbatchnorm_function_template, [conv2dbatchnorm_config_template, conv_mult_config_template], conv2dbatchnorm_include_list)
```

Comment:
Use the existing `Conv2D` templates, ie
`        self.register_templates('Conv2DBatchnorm'                 , conv2d_function_template,      [conv2d_config_template, conv_mult_config_template], conv2d_include_list)
`